src/main/java/sk/pocsik/services/api/HttpService.java

```diff
@@ -25,7 +29,7 @@ public HttpService() {
 
     public String findCardByName(String name) {
         StringBuilder url = new StringBuilder(this.apiUrl).append("?q=name:")
-                .append(name)
+                .append(URLEncoder.encode(name, StandardCharsets.UTF_8))
                 .append("*")
                 .append("&orderBy=name")
```

Comment:
Why you using an string builder?
Wouldnt you use support function for URLEncoded String?